### ISSUE TITLE
OPS not parsing log score

### ISSUE THREAD

**Issue by xKDHx** [autobrr/autobrr#1535]
**Version**
1.41.0

**Describe the bug**
Autobrr isn't parsing the log score for OPS releases. 

Announce:
[08:11] <&xxxxxx> TORRENT: Amorphous Androgynous – Tales Of Ephidrina – [2023] [Album] CD/FLAC/Lossless/Cue/Log/100 – ambient,electronic – https://xxxxxxx.xxxxxxx/torrents.php?id=xxxxxxx – https://xxxxxxx.xxxxxxx/torrents.php?id=xxxxxxx&torrentid=xxxxxxx&action=download

```
DBG
(OPS CD FLAC Test) for release: Amorphous Androgynous - Tales Of Ephidrina [2023] (CD FLAC Lossless) rejections: (wanted: perfect flac. got: [Cue FLAC Lossless Log], log score. got: 0 want: 100) method=CheckFilter module=filter
DBG
filter OPS CD FLAC Test rejected release: wanted: perfect flac. got: [Cue FLAC Lossless Log], log score. got: 0 want: 100 filter="OPS CD FLAC Test" indexer=ops module=release release="Amorphous Androgynous - Tales Of Ephidrina [2023] (CD FLAC Lossless)"
```
```
{
    "name": "OPS CD FLAC Test",
    "version": "1.0",
    "data": {
        "enabled": true,
        "max_downloads": 1,
        "max_downloads_unit": "EVER",
        "formats": [
            "FLAC"
        ],
        "quality": [
            "Lossless"
        ],
        "media": [
            "CD"
        ],
        "perfect_flac": true,
        "cue": true,
        "log": true,
        "log_score": 100,
    }
}
```

I tested with and without the `perfect_flac` filter, same result, it parses the log score as 0.

**Comment by zze0s** [comment]
Hey! OPS recently added log score and other values to the announce, so we need to update the parser to look for those values.

**Comment by zze0s** [comment]
Fixed in #1592
